healthcheck/health_check.py

```diff
@@ -13,10 +13,10 @@ def is_healthy():
         return jsonify({'status': 'UNHEALTHY'}), 503
 
 
-def start_health_endpoint():
-    app.run(host='0.0.0.0', port=5000)
+def start_health_endpoint(port):
+    app.run(host='0.0.0.0', port=port)
 
 
-def run():
-    t = Thread(target=start_health_endpoint, daemon=True)
+def run(options):
+    t = Thread(target=start_health_endpoint(options.port), daemon=True)
```

Comment:
I am not really sure if this is correct. According to Python's doc:
> _target_ is the callable object to be invoked by the run() method

but here you've passed `None` which is returned from the execution of `start_health_endpoint`.

One thing to do is to return a lambda that will be started by Thread, I think:
```python
def start_health_endpoint(port):
    return lambda port: app.run(host='0.0.0.0', port=port)

def run(options):
    t = Thread(target=start_health_endpoint(options.port), daemon=True)
```